cli/bin/grain.js

```diff
@@ -194,7 +201,9 @@ program
   .command("lsp <file>")
   .description("check a grain file for LSP")
   .action(
-    wrapAction(function (file, options, program) {
+    wrapAction(function (file) {
```

Comment:
I can't comment on the lines above the diff, but does this mean that the `run` command needs to be updated too?

Reply:
Nevermind, it looks like that isn't using `wrapAction`